src/app/screens/Home/index.tsx

```diff
@@ -18,12 +18,15 @@ import { useNavigate } from "react-router-dom";
 import { toast } from "react-toastify";
 import browser from "webextension-polyfill";
 import api from "~/common/lib/api";
+import utils from "~/common/lib/utils";
 import type { Allowance, Battery, Transaction } from "~/types";
 
 dayjs.extend(relativeTime);
 
 function Home() {
   const [allowance, setAllowance] = useState<Allowance | null>(null);
+  const [isBlocked, setIsBlocked] = useState<boolean | null>(null);
```

Comment:
```suggestion
  const [isBlocked, setIsBlocked] = useState<boolean>(false);
```

I think now we can always use a boolean and default to `false`